src/lib/site-import/status.js

```diff
@@ -70,16 +70,25 @@ export type ImportSqlCheckStatusInput = {
 async function getStatus( api, appId, envId ) {
 	const response = await api.query( {
 		query: IMPORT_SQL_PROGRESS_QUERY,
-		variables: { id: appId },
+		variables: { appId, envId },
 		fetchPolicy: 'network-only',
 	} );
 	const {
 		data: {
 			app: { environments },
 		},
 	} = response;
-	const { importStatus, jobs } = environments.find( e => e.id === envId );
-	const importJob = jobs.find( ( { type } ) => type === 'sql_import' );
+	if ( ! environments?.length ) {
+		throw new Error( 'Unable to determine import status from environment' );
+	}
+	const [ environment ] = environments;
+	const { importStatus, jobs } = environment;
```

Comment:
Will `importStatus` be `undefined` if there's no import currently in progress?

Reply:
No, it would only be undefined if there's never been an import for the site. Also it looks like it's not referenced in that case in a way that errors:
![Screen Shot 2021-01-31 at 6 57 49 PM](https://user-images.githubusercontent.com/1587282/106402163-46735400-63f6-11eb-9c54-c2f1b4731d74.png)

I _did_ notice a "Bad Request" error when the job doesn't match one in the system, but that's not unique to this branch.